app/assets/stylesheets/modules/subnavbar.scss

```diff
@@ -25,7 +25,15 @@
     color: $white;
   }
 
+  .sul-top-menu {
+    min-width: 300px;
```

Comment:
Could this  be done with padding instead?  Is 300px going to be correct for all zoom levels and screensizes?